File: C#/organulos.cs
using System;

public class Organulos {
	private int cantidadLisosomas;
	private int cantidadPeroxiomas;

	public int GetCantidadPeroxiomas() {
		return this.cantidadPeroxiomas;
	}
	public void SetCantidadPeroxiomas(ref int cantidadPeroxiomas) {
		this.cantidadPeroxiomas = cantidadPeroxiomas;
	}
	public int GetCantidadLisosomas() {
		return this.cantidadLisosomas;
	}
	public void SetCantidadLisosomas(ref int cantidadLisosomas) {
		this.cantidadLisosomas = cantidadLisosomas;
	}

	private Celula celula;

}

File: Python/organulos.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
import celula
from typing import List

class organulos(object):
	def getCantidadPeroxiomas(self) -> int:
		return self.___cantidadPeroxiomas

	def setCantidadPeroxiomas(self, aCantidadPeroxiomas : int) -> None:
		self.___cantidadPeroxiomas = aCantidadPeroxiomas

	def getCantidadLisosomas(self) -> int:
		return self.___cantidadLisosomas

	def setCantidadLisosomas(self, aCantidadLisosomas : int) -> None:
		self.___cantidadLisosomas = aCantidadLisosomas

	def __init__(self):
		self.___cantidadLisosomas : int = None
		self.___cantidadPeroxiomas : int = None
		self._unnamed_celula_ : celula = None
		"""# @AssociationKind Composition"""

